recipes/opencv/4.x/conanfile.py

```diff
@@ -192,6 +194,8 @@ def requirements(self):
             self.requires("ffmpeg/4.4")
         if self.options.parallel == "tbb":
             self.requires("tbb/2020.3")
+        if self.options.with_ipp == "intel-ipp":
+            self.requires("intel-ipp/2020")
```

Comment:
?

There is no intel-ipp recipe in CCI

Reply:
https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#how-to-package-libraries-that-depend-on-proprietary-closed-source-libraries